Jenkinsfile

```diff
@@ -133,28 +115,24 @@ def shouldDeployLongRunningInstance() {
 }
 
 def mvn(String args) {
-    sh "mvn ${args}"
+    sh "mvn --batch-mode ${args}"
```

Comment:
Why the addition of `--batch-mode`?

Reply:
It tells maven that we're not in an interactive shell, so it shouldn't ever prompt, or print fancy download bars that refresh or anything.